### PR TITLE
Adds command line sample for the Street View Publish API v1.

### PR DESCRIPTION
**Pull request by lbobryan**
I am adding a new command line sample for the Street View Publish API (https://developers.google.com/streetview/publish/) which shows how to upload a photo.

### CODE REVIEW

**Comment by lbobryan** [comment]
Friendly ping :).

**Comment by lbobryan** [comment]
Bump

File: streetview-publish-cmdline-sample/src/main/java/com/google/api/services/samples/streetview/publish/cmdline/StreetViewPublishSample.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.store.DataStoreFactory;
+import com.google.api.client.util.store.FileDataStoreFactory;
+import com.google.api.gax.core.FixedCredentialsProvider;
```
Comment (neozwu) [inline streetview-publish-cmdline-sample/src/main/java/com/google/api/services/samples/streetview/publish/cmdline/StreetViewPublishSample.java]: gax is pulled in as a transitive dependency. It's safer to declare a direct dependency if import directly.

Comment (lbobryan) [inline streetview-publish-cmdline-sample/src/main/java/com/google/api/services/samples/streetview/publish/cmdline/StreetViewPublishSample.java]: I have fixed this issue in the next commit.

File: streetview-publish-cmdline-sample/src/main/java/com/google/api/services/samples/streetview/publish/cmdline/StreetViewPublishSample.java [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+import com.google.auth.Credentials;
+import com.google.geo.ugc.streetview.publish.v1.*;
+import com.google.geo.ugc.streetview.publish.v1.StreetViewPublishResources.*;
+import com.google.geo.ugc.streetview.publish.v1.StreetViewPublishResources.Photo;
+import com.google.geo.ugc.streetview.publish.v1.StreetViewPublishResources.UploadRef;
+import com.google.geo.ugc.streetview.publish.v1.StreetViewPublishServiceGrpc;
+import com.google.protobuf.*;
```
Comment (lbobryan) [inline streetview-publish-cmdline-sample/src/main/java/com/google/api/services/samples/streetview/publish/cmdline/StreetViewPublishSample.java]: I have fixed this issue in the next commit.

**Pull request merged**